lib/axon/loop.ex

```diff
@@ -504,11 +504,37 @@ defmodule Axon.Loop do
       model
       |> Axon.Loop.trainer(loss_weights)
       |> Axon.Loop.run(data)
+
+  ## Options
+
+    * `:log` - training loss and metric log interval. Set to 0 to silence
+      training logs. Defaults to 50
   """
-  def trainer(model, loss, optimizer) do
+  def trainer(model, loss, optimizer, opts \\ []) do
```

Comment:
Another option is to have a `|> Axon.Loop.attach_default_log_handler`.

Reply:
Do you mean rather than adding logging every time, make it explicit:

```elixir
model
|> Axon.Loop.trainer(:mean_squared_error, :sgd)
|> Axon.Loop.metric(:mean_absolute_error)
|> Axon.Loop.attach_default_log_handler()
|> Axon.Loop.run(data)
```

Reply:
Yes. It is just a suggestion though. :)

Reply:
Actually, I prefer your current approach. Ship it!

Reply:
Most will want something like that by default and it is like 10 chars to opt out, so current approach is good!